core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7350,6 +7355,39 @@ components:
         payload_hex:
           type: string
           description: The hex-encoded system transaction payload. Only returned if enabled in TransactionFormatOptions on your request.
+    TransactionOutcomes:
+      description: |
+        For the given transaction, contains individual entity resource balance changes.
+        The balance changes accounts for the fee payments as well.
+        Current implementation does not take into account recalls, but this will change in a future update.
```

Comment:
Ohh both of these should be fixed now. Could this be fixed here/wherever you copy-pasted from?